urlapi/helpers.go

```diff
@@ -0,0 +1,47 @@
+package urlapi
+
+import (
+	"fmt"
+
+	"go.vocdoni.io/proto/build/go/models"
+)
+
+func (u *URLAPI) getProcessSummaryList(pids ...[]byte) ([]*ProcessSummary, error) {
+	processes := []*ProcessSummary{}
+	for _, p := range pids {
+		procInfo, err := u.scrutinizer.ProcessInfo(p)
+		if err != nil {
+			return nil, fmt.Errorf("cannot fetch process info (%v)", err.Error())
```

Comment:
just use `fmt.Errorf("message: %w", err)` with errors: simpler, and also supports error wrapping